### PR TITLE
Add coveralls support to Travis

### PR DESCRIPTION
**Pull request by abarth**
This patch causes us to upload a coverage report to coveralls.io.

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
00:00 +0: loading /home/travis/build/flutter/flutter/packages/flutter/test/animation/animation_controller_test.dart
Shell: Observatory listening on http://127.0.0.1:19972
Shell: Diagnostic server listening on http://127.0.0.1:37811

Are those logs new from your patch?  Do you want the diagnostic server running?

**Comment by abarth** [comment]
> Are those logs new from your patch?

Yes.

> Do you want the diagnostic server running?

No.

It's a work in progress.

File: dev/bots/test.sh [hunk 1]
```diff
@@ -23,8 +23,13 @@ flutter analyze --flutter-repo
 (cd dev/automated_tests; ! flutter test test_smoke_test/fail_test.dart > /dev/null)
 (cd dev/automated_tests; flutter test test_smoke_test/pass_test.dart > /dev/null)
 
+COVERAGE_FLAG=
+if [ -n TRAVIS ]; then
```
Comment (Hixie) [inline dev/bots/test.sh]: `$TRAVIS`

**Comment by abarth** [comment]
https://coveralls.io/github/flutter/flutter has the data

**Comment by Hixie** [comment]
LGTM

**Pull request merged**